rules/integrations/beaconing/command_and_control_beaconing.toml

```diff
@@ -0,0 +1,50 @@
+[metadata]
+creation_date = "2023/09/22"
+integration = ["beaconing"]
+maturity = "production"
+min_stack_comments = "New rule"
+min_stack_version = "8.10.1"
```

Comment:
This package never had rules in it since we were already planning to migrate them out. The first release of the package  went out a week ago with min stack version 8.10.1.